Update all documentation for 2.7.0 (#7533)
keepassxreboot/keepassxc: docs/topics/UserInterface.adoc

*(A) Groups* - Organize your entries into discrete groups to bring order to all of your sensitive information. Groups can be nested under each other to create a hierarchy. Settings from parent groups get applied to their children. You can hide this panel on the View menu.

*(B) Tags* - Dynamic groups of entries that can be quickly displayed with one click. Any number of custom tags can be added when editing an entry. This panel also includes useful pre-defined searches, such as finding expired and weak passwords.

> *\(C) Entries* - Entries contain all the information for each website or application you are storing in KeePassXC. This view shows all the entries in the selected group. Each column can be resized, reordered, and shown or hidden based on your preference. Right click the header row to see all available options.

Reviewer: ```suggestion
*\(C) Entries* - Entries contain all the information you want to store for a website or application in KeePassXC. This view shows all the entries in the selected group. Each column can be resized, reordered, and shown or hidden based on your preference. Right-click the header row to see all available options.

```

.Compact mode comparison
image::compact_mode_comparison.png[]

=== Screenshot Security

> By default, KeePassXC prevents recording and screenshots of the application window on Windows and macOS. This prevents inadvertent spillage of information during meetings and disallows other applications from being able to capture the window contents. If you would like to enable screencapture, you must start the application with the `--allow-screencapture` command line flag.

Reviewer: ```suggestion
By default, KeePassXC prevents recordings and screenshots of the application window on Windows and macOS. This prevents inadvertent spillage of information during meetings and disallows other applications to capture the window contents. If you would like to enable screen capture, you must start the application with the `--allow-screencapture` command line flag.
```
